ci/test_cudf_polars_polars_tests.sh

```diff
@@ -27,8 +27,6 @@ git clone https://github.com/pola-rs/polars.git --branch "${TAG}" --depth 1
 # Install requirements for running polars tests
 rapids-logger "Install polars test requirements"
 rapids-pip-retry install -r polars/py-polars/requirements-dev.txt -r polars/py-polars/requirements-ci.txt
-# TODO: Workaround until https://github.com/pola-rs/polars/issues/21274 is fixed.
-rapids-pip-retry install connectorx==0.4.1
```

Comment:
We will need this, AFAIK CI was failing even if we ignore the following tests.

Reply:
No we can remove it. It was failing because I didn't have the correct name of the test to skip

Reply:
Before I had (missed the extra forward slash)
```
tests/unit/io/database/test_read.py::test_read_database_cx_credentials[fakedb:/123:456@account/database/schema?warehouse=warehouse&role=role]
```
But the name was
```
tests/unit/io/database/test_read.py::test_read_database_cx_credentials[fakedb://123:456@account/database/schema?warehouse=warehouse&role=role]
```

Reply:
Okay, approved. Shall we wait for the CI to complete to test this change?

Reply:
CI passed on this test.

Reply:
> Okay, approved. Shall we wait for the CI to complete to test this change?

Yeah sure. Might help others be more comfortable with merging it :)